### PR TITLE
Registration-related modifications

### PR DESCRIPTION
**Pull request by cobusc**

### CODE REVIEW

File: girleffect_oidc_integration/auth.py [hunk 1]
```diff
@@ -12,6 +12,22 @@
...
+    This function is called on registration (new user) as well as login events.
+    This function provides the mapping from the OIDC claims fields to the
+    internal user profile fields.
+    :param user: The user profile
+    :param claims: The claims for the profile
+    """
+    print(claims)
+    user.first_name = claims.get("given_name") or claims["nickname"]
+    user.last_name = claims.get("family_name") or ""
+    user.email = claims.get("email")  # Email is optional
```
Comment (ruands) [inline girleffect_oidc_integration/auth.py]: What happens when there is no "email" in `claims`? Should there not also be an `or ""`?

File: wagtail_client/templates/wagtail_client/home.html [hunk 2]
```diff
@@ -7,23 +7,46 @@
...
-            <h1>Welcome back!</h1>
+            <h1>Welcome {{ user.first_name }}!</h1>
             <h1>email: {{ user.email }}</h1>
-            <h1>{{ user }}</h1>
             <div class="submit">
                 <a class="button" href="{% url "logout" %}">Logout</a>
             <div>
+            <br />
+            <div class="submit">
+                <a class="button" href="http://172.18.0.3:8000/profile/edit/?redirect_url=http://172.18.0.4:8000/">Edit Profile</a>
```
Comment (ruands) [inline wagtail_client/templates/wagtail_client/home.html]: This is the demo so I don't think we need to worry about trans tags here? I which case this is fine.

Comment (cobusc) [reply wagtail_client/templates/wagtail_client/home.html]: @ruands Yes. I have not done any `trans` tags as this is for demo purposes only.

**Comment by ruands** [comment]
Just that one, and a question. Else +1

File: wagtail_client/templates/wagtail_client/home.html (hunk 2 above)
Comment (ruands) [reply wagtail_client/templates/wagtail_client/home.html]: Happy with that 👍

**Pull request merged**